src/flight_booking_agent/agents/information_gatherer.py

```
@@ -1,39 +1,117 @@
+    )
+    return_date: Optional[str] = Field(
+        description="Ngày về cho chuyến bay khứ hồi (nếu có). Cần chuyển đổi thành định dạng YYYY-MM-DD."
+    )
+    passenger_count: Optional[int] = Field(
+        description="Tổng số lượng hành khách. Chỉ lấy số nguyên từ câu nói, ví dụ '2 người' -> 2."
+    )
+    # Bạn có thể thêm các trường tùy chọn khác ở đây, ví dụ: travel_class
+
+def convert_relative_date(date_str: str) -> str:
+    """Hàm phụ trợ để chuyển đổi 'ngày mai', 'hôm nay' thành YYYY-MM-DD."""
+    if not date_str: return None
```

Ruff (0.13.2)

Multiple statements on one line (colon)

(E701)